README.md

```diff
@@ -117,8 +122,12 @@ docker run -d --name influxdb \
       -v $PWD/deploy/initdb.sh:/docker-entrypoint-initdb.d/initdb.sh \
       influxdb:1.8
 ```
+      
+      
+   NOTE: NuoDB Collector only supports InfluxDB version 1.X.
```

Comment:
nit: this sentence may format better if lined up with other sub statements like, "Use ..." and "All NuoDB components..." above.